amundsen-kube-helm/templates/helm/neo4j/templates/neo4j-s3-backup.yaml

```diff
@@ -0,0 +1,46 @@
+{{ if and (and .Values.neo4j.backup.enabled (eq .Values.provider "aws")) (or (kindIs "invalid" .Values.neo4jpvc) .Values.neo4jpvc) }}
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+    name: neo4j-s3-backup
+spec:
+    schedule: {{ .Values.neo4j.backup.schedule }}
+    jobTemplate:
+        spec:
+            template:
+                spec:
+                    containers:
+                        - name: backup-neo4j
+                          image: mesosphere/aws-cli:latest
+                          command:
+                              - "/bin/sh"
+                              - "-c"
+                              - |
+                                  apk -v --update add --no-cache --quiet curl
+                                  BACKUP_NAME="graph.db-backup-$(date "+%Y-%m-%d-%H:%M:%S").csv"
+                                  curl --user neo4j:test -H accept:application/json -H content-type:application/json \
+                                    -d '{"statements":[{"statement":"CALL apoc.export.csv.all(\"\/var\/lib\/neo4j\/data\/'$BACKUP_NAME'\", {})"}]}' \
```

Comment:
@javamonkey79 
Is exporting csv sufficient for backup? I am asking because we also back up schema (`CALL apoc.export.cypher.schema`) along with data backup via `CALL apoc.export.graphml.all`

(We use `graphml` backup as it showed the best performance at that time which was more than a year ago)

May be csv export does not require schema backup?

Reply:
hey @jinhyukchang , yes, I think csv based backup will work you would just have to use [import csv](https://neo4j.com/docs/labs/apoc/current/import/import-csv/) on the other side.

That said, if graphml backup is more performant, can you share your cypher commands and I can try it?

Reply:
@jinhyukchang do you have a command that works better than csv that you can share? If not, can we merge this?

Reply:
@javamonkey79 Sorry, I missed your response.

Here's APOC calls: one for schema and the other for data. Let me know 😄 

` echo "CALL apoc.export.cypher.schema(${schema_file}, {});" | ( time ${NEO4J_BIN}/neo4j-shell - ) | tee -a ${BACKUP_LOG_FILE}
`

` echo "CALL apoc.export.graphml.all(${data_file}, {useTypes: true, readLabels: true});" | ( time ${NEO4J_BIN}/neo4j-shell - ) | tee -a ${BACKUP_LOG_FILE}
`

Reply:
super, thanks @jinhyukchang ! I've updated accordingly and tested it out on our QA cluster.